### ISSUE TITLE
Ignore 'suppressed' assemblies?

### ISSUE THREAD

**Issue by alvanuffelen** [ncbi/datasets#216]
Dear Eric

Is there a way to ignore assemblies that are suppressed (related to #111 )?
I used following command:
`datasets summary genome taxon 1496 --report ids_only --exclude-atypical --assembly-source RefSeq --assembly-level complete --assembly-version latest`
This gives me 118 assemblies. Unfortunately, [one assembly](https://www.ncbi.nlm.nih.gov/assembly/GCF_001447175.1/) is 'suppressed' because of 'many frameshifted proteins'.

**Comment by alvanuffelen** [comment]
Hi Eric

Thanks for the explanation!
Would it be interesting to implement a flag to ignore 'suppressed' assemblies such that post-processing is not necessary?

**Comment by ericcox1** [comment]
>Would it be interesting to implement a flag to ignore 'suppressed' assemblies such that post-processing is not necessary?
Yes, this is a good suggestion. We are going to think about the best way to implement this and I'll try to comment on this thread when we have an update.

Thanks again for your feedback.

**Comment by ericcox1** [comment]
Hi Alvanuffelen,

Thanks again for your suggestion. We have decided to not return suppressed assemblies when querying by taxon. In other words the following command:

`datasets summary genome taxon 1496 --report ids_only` will no longer return the suppressed genome assembly, GCF_001447175.1.

This update is now in production.

Note that you can still get get data for suppressed assemblies by requesting them by accession, for example:
`datasets summary genome accession GCF_001447175.1`

Best,
Eric
